packages/assertion-monitor/reportAssertionMonitorAlertToSlack.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv'
+import { postSlackMessage } from '../utils/postSlackMessage'
+
+dotenv.config()
+
+const slackToken = process.env.BATCH_POSTER_MONITORING_SLACK_TOKEN
+const slackChannel = process.env.BATCH_POSTER_MONITORING_SLACK_CHANNEL
```

Comment:
Ooops, we need new env tokens for assertion monitoring :) 
`ASSERTION_MONITORING_SLACK_TOKEN` and `ASSERTION_MONITORING_SLACK_CHANNEL`